src/globus_sdk/experimental/login_flow_manager/command_line_login_flow_manager.py

```diff
@@ -111,22 +103,34 @@ def run_login_flow(
             to the authentication flow to control how the user will authenticate.
         """
         authorize_url = self._get_authorize_url(auth_parameters, self.redirect_uri)
-        auth_code = self._print_and_prompt(authorize_url)
+        self.print_authorize_url(authorize_url)
+        auth_code = self.prompt_for_code()
 
         return self.login_client.oauth2_exchange_code_for_tokens(auth_code)
 
-    def _print_and_prompt(self, authorize_url: str) -> str:
-        # Prompt the user to authenticate
+    def print_authorize_url(self, authorize_url: str) -> None:
+        """
+        Prompt the user to authenticate, using the computed ``authorize_url``.
+
+        :param authorize_url: The URL at which the user will login and consent to
+            application accesses.
+        """
+        login_prompt = "Please authenticate with Globus here:"
         print(
             textwrap.dedent(
                 f"""
-                {self.login_prompt}
-                {"-" * len(self.login_prompt)}
+                {login_prompt}
+                {"-" * len(login_prompt)}
                 {authorize_url}
-                {"-" * len(self.login_prompt)}
+                {"-" * len(login_prompt)}
                 """
             )
         )
 
-        # Request they to enter the auth code
-        return input(f"{self.code_prompt}\n").strip()
+    def prompt_for_code(self) -> str:
+        """
+        After the authorization URL is printed, ask the user to enter the
+        authorization code.
```

Comment:
If we want to express the order in which this method will be executed relative to other operations, I wouldn't do it inside of the method (i.e., nothing internal to the method interacts with the printing of the authorization URL or controls ordering). I would advocate that we address either in `run_login_flow` or at the class level, but I can't place suggestions at either location via GH.

```suggestion
        Prompt the user to enter an authorization code.
```

Reply:
I'm fine with this rephrase. In concert with the `click` example (which I think we'll move or copy from the changelog at some point), the usage mode will be clear.